baselines/disable-packing-test/src/v1beta1/echo_client.ts.baseline

```diff
@@ -607,6 +607,11 @@ export class EchoClient {
     this.initialize();
     return this.innerApiCalls.wait(request, options, callback);
   }
+  async waitDecodeLRO(name: string){
```

Comment:
Let's brainstorm on the better name with @bcoe and @JustinBeckwith. Nobody really knows what's an LRO :)

Reply:
Heh - what does this actually do?  Can you provide a few practical examples of what it's usage would be like, and why I'd do it?

Reply:
Sure, basically we want to create an interface for users to decode their LRO, related issue is here from speech: https://github.com/googleapis/nodejs-speech/issues/399.

Currently they have  a way to do that:
```
const [operation] = await speech.longRunningRecognize(request);
const decodedOperation = new gax.Operation(getResponse /* */, speech._descriptors.longrunning.longRunningRecognize, gax.createDefaultBackoffSettings());
console.log(decodedOperations.name)
console.log(decodedOperations.done)
console.log(decodedOperations.result)
console.log(decodedOperations.metadata)
```
But it might be too difficult for users to use some undocumented private members, so we decided to provide them a method `decodeLRO` which will do the same job. 
Micro-gen issue: https://github.com/googleapis/gapic-generator-typescript/issues/170
```
const [operation] = await speech.longRunningRecognize(request);
const decodedOperation = decodeLRO(operation.name);
```

Reply:
Sorry for being difficult - taking many steps back 🤣  I read through the original issue, and it starts with this:

```js
const operation = await speechClient.operationsClient.getOperation({ name });
```

I think the original problem is that this returns a serialized proto buffer.  Having such a public method directly return a serialized protobuf doesn't quite feel right to me.  Do we have this pattern in other places?

I guess my question is - why are we asking the user to jump through the decoding hoop here at all?

Reply:
I explained the use case below :)  The method we're going to add here won't return any protobuf, just the real answer.

@xiaozhenliu-gg5 Maybe it's better if you provide code samples here?

Reply:
Yes sure, I will add in-code sample for this piece. Thanks!